### DISCUSSION TITLE
Export Shopping List to Google Keep?

### DISCUSSION THREAD

**Discussion by TimNiceButDim** [TandoorRecipes/recipes#3180] [Ideas]
Hi there. Firstly thanks for the brilliant app. Me and my partner have been enjoying adding all of our recipes over the last few days.

Something that would be really handy would be the ability to export the shopping list. In particular to export it to Google Keep.

Is this something that could be added in the future?

My first ever post on GitHub so hope I've done this right!

Cheers.

**Comment by smilerz** [comment]
The shopping list can be exported to csv, does Keep support importing a list that way?

**Reply by TimNiceButDim** [reply]
I don't believe so, no.

**Comment by vabene1111** [comment]
google keep works with makrdown export, it just needs a list thats row seperated
